src/components/form/FormField.js

```diff
@@ -71,6 +75,7 @@ const Types = {
   [FieldTypeOptions.REFERENCE_NUMBER]: 'text',
   [FieldTypeOptions.STRING]: 'text',
   [FieldTypeOptions.TEXTAREA]: 'text',
+  [FieldTypeOptions.FRACTIONAL]: 'text',
```

Comment:
I'm not sure if this does anything, though I don't fully understand the function of this enum in the first place. It seems to be only used to find out which value to pass down to the specific form field component as the `type` prop, but `FieldTypeFractional` doesn't make use of that prop. The fields that were previously defined here all already have "text" as the default value for that prop or don't use it at all either, so it's more or less of a noop?

Reply:
yah. can be removed.